scripts/lib/sign-win32.ts

```diff
@@ -30,9 +30,11 @@ const REQUIRED_WINDOWS_CONFIG = {
 const DEFAULT_WINDOWS_CONFIG = {
   certificateSha1:        '', // set via CSC_FINGERPRINT
   rfc3161TimeStampServer: 'http://timestamp.digicert.com',
+  extraFiles:             [] as Array<string>,
```

Comment:
Where is this field used? I pulled it out and could run `npm run sign -- "dist/Rancher Desktop.some-version.zip` without a problem.

Reply:
I had some other code to try to munge it, but it didn't workout and I missed removing this change.

Amended the commit to get rid of it.